### PR TITLE
Make the scroll amount configurable

### LINKED ISSUE #379: Increase scroll amount / make it configurable
I don't know if I am the only one, but I find the amount of scroll in `go flutter` too low in comparison to any other desktop / web application. I found in the source code that currently `go-flutter` is using `50` as the amount to scroll. 
https://github.com/go-flutter-desktop/go-flutter/blob/f67b5ecc81024ff9adb9abe5d06d2667e6130007/glfw.go#L191

I compared how much a simple ListView using Flutter web is scrolling and it looks that is a flat `100`.
I think it would be great if that amount could be increased or better yet, make it configurable with a constant. At the moment, at least for me, scrolling in a go-flutter app doesn't feel great.

This is the small code I tried to compare Flutter web and `go-flutter`.

```dart
void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Page(),
    );
  }
}

class Page extends StatelessWidget {
  const Page({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Scrollbar(
        child: ListView.builder(
          itemCount: 50,
          shrinkWrap: true,
          itemBuilder: (context, i) {
            return Container(
              alignment: Alignment.centerLeft,
              height: 100,
              color: i % 2 == 0 ? Colors.blue : null,
              child: Text("Item $i"),
            );
          },
        ),
      ),
    );
  }
}
```

### PR DESCRIPTION
**Pull request by davidmartos96**
Issue #379 

The scroll amount remains unchanged. The PR makes the amount configurable with the config struct.

### CODE REVIEW

**Comment by jld3103** [comment]
Hm if the scroll amount is wrong compared to the official flutter implementations why don't change it to the official scroll amount? But that could still be achieved with your PR.

**Comment by davidmartos96** [comment]
Well, I made it that way in case you care about breaking changes in the behavior of the existing apps. 
If you decide to simply change the current scroll amount, just let me know and I will update the PR

**Comment by jld3103** [comment]
I don't think anyone ever tried fixing the scroll in their app. So I guess it would fix it for everyone. @pchampio

File: option.go [hunk 1]
```diff
@@ -53,6 +54,7 @@ var defaultApplicationConfig = config{
 	keyboardLayout:    KeyboardQwertyLayout,
 	windowMode:        WindowModeDefault,
 	windowAlwaysOnTop: false,
+	scrollAmount:      50.0,
```
Comment (pchampio) [inline option.go]: As suggested, let's set the defualt `scroolAmount` to `100.0`

Comment (davidmartos96) [reply option.go]: Ok, I agree.

**Review by pchampio — APPROVED** [review]
Thanks for the contribution!
This will be available in the next version,
use `hover run -b '@master'` for now.

**Comment by davidmartos96** [comment]
@pchampio  Ok, great! Thanks!

**Pull request merged**